bootstrap.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+GIT_ROOT=https://github.com/jeffwarnica/CloudForms_Essentials
```

Comment:
Should this be `https://github.com/ramrexx/CloudForms_Essentials` when it's merged?